.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,26 @@
+---
+name: Bug report
+about: Report a bug.
```

Comment:
```suggestion
description: Report a bug.
```

Reply:
As per this [doc ](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/manually-creating-a-single-issue-template-for-your-repository), yml  files should have  a description.